androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/screen/surveydetail/SurveyQuestionContent.kt

```diff
@@ -0,0 +1,189 @@
+            )
+            Box(
+                modifier = Modifier
+                    .matchParentSize()
+                    .background(
+                        brush = Brush.verticalGradient(
+                            colors = listOf(
+                                Color.Black.copy(alpha = TopGradientAlpha),
+                                Color.Black.copy(alpha = BottomGradientAlpha)
+                            )
+                        )
+                    )
+            )
+        }
+
+        CloseButton(
+            modifier = Modifier
+                .statusBarsPadding()
+                .align(Alignment.TopEnd)
+                .padding(vertical = 20.dp, horizontal = 20.dp),
+            onClick = onCloseClick
+        )
+    }
+
+    HorizontalPager(
+        pageCount = surveyQuestionUiModels.size,
+        state = pagerState,
+        userScrollEnabled = false,
+        modifier = Modifier
+            .fillMaxSize()
+    ) { page ->
+        QuestionContent(
+            page + 1,
+            surveyQuestionUiModels.size,
+            surveyQuestionUiModels[page],
+            onNextClick = {
+                scope.launch {
+                    pagerState.animateScrollToPage(page + 1)
+                }
+            },
+            modifier = Modifier.fillMaxSize()
+        )
+    }
+}
+
+@Composable
+private fun QuestionContent(
+    page: Int,
+    count: Int,
+    question: SurveyQuestionUiModel,
+    onNextClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier
+            .statusBarsPadding()
+            .navigationBarsPadding()
+            .fillMaxWidth()
+            .padding(top = 70.dp)
+            .padding(horizontal = 20.dp)
+    ) {
+        Text(
+            text = "$page/$count",
+            color = White.copy(alpha = 0.50f),
+            style = MaterialTheme.typography.body2,
+            modifier = Modifier.padding(top = 32.dp)
+        )
+
+        Text(
+            text = question.text,
+            color = White,
+            style = MaterialTheme.typography.h4,
+            modifier = Modifier.padding(top = 8.dp)
+        )
+
+        Box(
+            modifier = Modifier
+                .fillMaxWidth()
+                .weight(1f)
+        ) {
+            AnswerContent(
+                question = question,
+                modifier = Modifier.align(Alignment.Center)
+            )
+        }
+
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(bottom = 54.dp),
+            horizontalArrangement = End
+        ) {
+            FloatingActionButton(
+                backgroundColor = White,
+                onClick = onNextClick,
+                modifier = Modifier.size(56.dp)
+            ) {
+                Image(
+                    painter = painterResource(id = R.drawable.ic_arrow_right),
+                    contentDescription = null
+                )
+            }
+        }
```

Comment:
We can use `align` for alignment. Don't need to wrap inside another view here.
```suggestion
        FloatingActionButton(
            backgroundColor = White,
            onClick = onNextClick,
            modifier = Modifier.size(56.dp).align(Alignment.End)
        ) {
            Image(
                painter = painterResource(id = R.drawable.ic_arrow_right),
                contentDescription = null
            )
        }
```

Reply:
For some reason adding an end padding makes the button narrower. However, used `Box` instead for the same purpose in this commit: https://github.com/nimblehq/avishek-phong-kmm-ic/pull/120/commits/a9b678ce651f4a43e25c7334658eea45de84039a

Reply:
@AVI5HEK I think the sequence of applying `align`, `size` and `padding` matters. Can you please try applying like the following. I tried it and it turns out the same.
```
modifier = Modifier
                    .padding(end = 20.dp)
                    .align(Alignment.End)
                    .size(56.dp)
```

Reply:
@kaungkhantsoe you're right. Worked like charm. TIL that the order matters. Only the `size` needs to be the last. The order of `padding` and `align` doesn't matter as it seems.